rules-tests/DeadCode/Rector/Property/RemoveUnusedPrivatePropertyRector/Fixture/only_read.php.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector\Fixture;
+
+class OnlyRead
+{
+    private $unused;
```

Comment:
beware: when the property has a default value, the code within the if is not dead and should be left untouched